pkg/controlplane/control/manager.go

```diff
@@ -193,10 +233,87 @@ func (m *Manager) AddImport(ctx context.Context, imp *v1alpha1.Import) (err erro
 	return m.addImportService(ctx, imp, userService)
 }
 
+// deleteClDataplane deletes pod to ipToPod list.
+func (m *Manager) deleteEndpoint(esID types.NamespacedName) {
+	if strings.Contains(esID.Name, dpapp.Name) {
```

Comment:
This seems hacky.
I think you should instead check of esID.Name is found in m.endpoints (see my previous comment on making m.endpoints indexed by esname).

Reply:
I plan to remove this, since it doesn't do anything